components/applicationsets/kustomization.yaml

```diff
@@ -5,3 +5,4 @@ kind: Kustomization
 resources:
   - core-components-appset.yaml
   - clusters-appset.yaml
+  - worker-nodes-appset.yaml
```

Comment:
What if we *don't* want to add a worker node?  I.e. a 3-node compact cluster?  Does this apply?

Or will action only be taken if /add-workers/<node> is populated / pushed?

Reply:
Agreed. Note that this is a default manifest that runs everything. That should not be the way to launch this. Note that similar thing can happen with clusters-appset.yaml since it may trigger the  launch of multiple clusters.